src/main/java/org/gitlab4j/api/models/Badge.java

```diff
@@ -0,0 +1,84 @@
+package org.gitlab4j.api.models;
+
+import org.gitlab4j.api.utils.JacksonJson;
+
+public class Badge {
+
+  private Integer id;
+  private String link_url;
+  private String image_url;
+  private String rendered_link_url;
+  private String rendered_image_url;
+  private String kind;
+
```

Comment:
All variable and method in this file should be camel cased.

Reply:
Sorry, I was not sure if I should follow the json variables from GitLab api or not. Fixed.